scrapers/VRBangers.yml

```
@@ -25,16 +26,18 @@ xPathScrapers:
         selector: //div[contains(@class,"second-text")]/div//text()
         concat: " "
       Tags:
-        Name: //div[starts-with(@class,"video-item__tags ")]//a/text()
+        Name: //a[contains(@href,"category/")]/text()
```

This need to be more specific.
In all of the sites category anchors are sometimes used in the description with different text that shouldnt be matched
eg
`https://vrconk.com/video/pocahontas-a-xxx-parody/` , `https://vrbangers.com/video/yoga-hosers/` , `https://vrbgay.com/video/desperate-housedudes/` , `https://vrbtrans.com/video/the-best-cumshot-compilation/`.
Notice the anchors in the description and the tags you get when scraping...
The following seems to work (limited testing though )
```suggestion
        Name: //div[@data-testid="video-categories-list"]/a[contains(@href,"category/")]/text()
```